docs/user-documentation/oai.md

```diff
@@ -17,6 +17,23 @@ Enabling Islandora OAI-PMH provides:
 - [default configuration](http://localhost:8000/admin/config/services/rest/oai-pmh) to connect this view with the REST OAI-PMH module.
 - the ability to define additional sets by building additional views. Additional sets can be created by making views with the Entity Reference view display mode and enabling them on the rest_oai_pmh configuration page: /admin/config/services/rest/oai-pmh.
 
-The OAI-DC metadata format is aided by the Islandora OAI-PMH module to include the RDF Mapping for the linked agent field (from Islandora Defaults). Including agent links in the OAI-PMH metadata currently requires updating the RDF mapping to include a Dublin Core predicate for that field or any other additional fields. Alternatively, the rest_oai_pmh module also supports defining mappings with the metatag module or creating a custom metadata profile using the Twig templating system.
-
 The rest_oai_pmh module indexes set membership, so repository items may not appear immediately in their respective sets. Indexing will happen automatically during cron runs but can be triggered manually at `/admin/config/services/rest/oai-pmh/queue`.
+
+## OAI-DC Metadata Format/Metadata Mappings
+When you look at the [default OAI-PMH configuration](http://localhost:8000/admin/config/services/rest/oai-pmh), you will see that the "OAI Dublin Core (RDF Mapping)" is used to produce the oai_dc metadata format. 
+
+Note that this is not a separate RDF mapping. It uses the RDF mapping for the included content type(s) mapping (e.g. rdf.mapping.node.islandora_object.yml) and filters out any fields where the predicate is not in the Dublin Core namespace.
+
+!!! tip "Field values not showing up in OAI-DC record?"
+    If you want the value of a field to be emitted in the OAI-DC record, you must assign a Dublin Core predicate for that field in your content type's RDF mapping. If you are wondering why a field is not showing up in the OAI-DC record, the content type's RDF mapping is the first thing to check. 
+
+The OAI-DC metadata format is aided by the Islandora OAI-PMH module to include the RDF Mapping for the linked agent field (from Islandora Defaults). Including agent links in the OAI-PMH metadata currently requires updating the RDF mapping to include a Dublin Core predicate for that field or any other additional fields.
+
+## Creating additional metadata formats
+This typically involves creating a new plugin.
+
+The [Drupal rest_oai_pmh module's DefaultMap plugin](https://git.drupalcode.org/project/rest_oai_pmh/-/blob/2.0.x/src/Plugin/OaiMetadataMap/DefaultMap.php) provides a basic model to follow for creating a plugin. 
+
+Exact implementation of your plugin will depend on your data model. If your data model uses anything like Typed Relation field types, Paragraphs, or other complex nested entity modeling, you will need to add custom logic to build the values to emit via OAI-PMH for those fields.
```

Comment:
Not sure the best way to present / convey this for documentation but a lot of the default `rest_oai_pmh` conventions expect a very flat render array style of fields to be returned when implementing [`transformRecord()`](https://git.drupalcode.org/project/rest_oai_pmh/-/blob/2.0.x/src/Plugin/OaiMetadataMap/DublinCoreRdf.php#L116) unless you are really going to go off the beaten path with introducing a custom twig template or build method.

Guess tl;dring the above would be the default processes expect a flat array of fields to be returned?

Reply:
Yeah, I think anyone with any experience writing any code for Islandora will get the drift. I feel like I even ran into this basic idea writing field display stuff for I7.  Tweaked the wording a bit.